docker/LICENSE

```diff
@@ -0,0 +1,191 @@
+
```

Comment:
Fair point.  I don't see us wanting to change the license, so I can drop that file.